### PR TITLE
Implement the interactive list feature

### PR DESCRIPTION
**Pull request by bruk19**
1. Added the logic to toggle each task's completed status
2. Added the onchange handler to the clear all completed link to remove the completed tasks from the array
3. Implemented the localStorage update logic after switching the task's status
4. Fixed linter errors

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @bruk19,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no description in this PR, please add a description to the Pull Request 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/addTaskUI.js [hunk 1]
```diff
@@ -16,21 +16,17 @@ const appendTask = (item, list) => {
       moveImg.src = trashSrc;
```
Comment (elmar8287) [inline src/addTaskUI.js:16]: - [ ] Please, consider implementing the `filter()` method for the "Clear all completed" button, as it is mentioned in the project requirements. As I understand, you are using `splice` from "clearTask" function 
Screen Shot 2022-06-03 at 10 01 08
 
> clearTask function
Screen Shot 2022-06-03 at 10 04 07

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

To highlight:

- No linter issues
- Nice design
- Informative PR 👍 

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/remove.js [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-    const nextCheck = document.querySelector(`#task-${i + 2}`);
-    nextCheck.id = `task-${i + 1}`;
+    /*
+    let ind = i + 2;
+   let nextCheck = document.querySelector(`#task-${ind}`)
+    nextCheck.attributes.id = `task-${i + 1}`;
     nextCheck.value -= 1;
     list[i].index -= 1;
     localStorage.setItem('tasks', JSON.stringify(list));
+    */
```
Comment (RolandM99) [inline src/remove.js:18]: - [ ] [ OPTIONAL ] To make your code clear, I'll suggest you remove the commented code from your code. Kindly, remove them.

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @bruk19 , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**